### ISSUE TITLE
Edits seem to be grouped into changesets incorrectly

### ISSUE THREAD

**Issue by westnordost** [OSMCha/osmcha-frontend#265]
## *Brief Description*
Edits done on the same object are grouped into one changeset in the display, even if these changes have been done in different changesets.

## What  is the current behaviour?

Let's look at [changeset 56188366](https://osmcha.mapbox.com/changesets/56188366):
According to Osm Changeset Analyzer, the user added four tags on the **Heidbusch** road in that changeset:
![heidbusch-osmcha](https://user-images.githubusercontent.com/4661658/36062773-256944c6-0e73-11e8-9ab7-6f0d967374cd.png)

However, looking at the history of the element in JOSM, one can see that the four changes were actually uploaded in three different changesets (i.e. look at the changeset number I framed with a green rectangle in the screenshot)
![heidbusch-josm](https://user-images.githubusercontent.com/4661658/36062787-839137f2-0e73-11e8-8b29-b738ed4a4d7c.png)

The [changeset on OpenStreetMap](https://www.openstreetmap.org/changeset/56188366#map=17/51.46089/6.94899) also lists only version 10 of the way as being part of the 56188366 changeset:
![heidbusch-osm-history](https://user-images.githubusercontent.com/4661658/36062808-e833ab90-0e73-11e8-9a06-7586ce340331.png)

## What  is the expected behaviour?

The expected behavior would be that OsmCha only shows the changes that really belong to the changeset that is currently shown

## Other Information / context:
This bug was initially reported at https://github.com/westnordost/StreetComplete/issues/853 but it turned out it is only not displayed correctly.

**Comment by matkoniecz** [comment]
Also reported as #492

**Comment by riQQ** [comment]
This should be fixed with #772, right?

**Comment by jake-low** [comment]
Yes, #722 fixed this. However, there is currently a logic bug in the [osmx-adiff-builder](https://github.com/OSMCha/osmx-adiff-builder/) which can still cause certain edits to be incorrectly grouped. The specifics are slightly different from the bug reported here, since it's a brand new codebase, but the effect is similar: certain edits may appear in changesets that they don't belong in. I'm tracking that bug in osmcha/osmx-adiff-builder#1.

I'll go ahead and close this bug, since the root cause was in a component of the OSMCha backend that is no longer in use. But be aware that a superficially similar bug still exists. I'm actively working on a fix, so that bug will hopefully also be fixed soon.
